### PR TITLE
Add a Help page to help direct users in need of assistance.

### PR DESCRIPTION
**Pull request by brian-brazil**
@juliusv

### CODE REVIEW

**Comment by grobie** [comment]
Any specific reasons not to use markdown? Should help to decouple content from presentation (like all the css classes a content author shouldn't need to know about.

File: content/help.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Help
+---
+<div class="col-md-8 col-md-offset-2 doc-content">
+  <h1>Help</h1>
+  <p>
+  If you are having a problem using Prometheusm, there are several ways to get
```
Comment (juliusv) [inline content/help.html]: Prometheusm -> Prometheus

**Comment by juliusv** [comment]
Hmm, this is duplicating so much of the "COMMUNITY" page. Also, we have to use the top menu sparingly, and adding another distinct nav item for "HELP" makes it look noticeably more blown up (harder for the eye to scan it).

Could some of this information be put into the COMMUNITY page somehow? Right now HELP is basically pointing people to (information in) COMMUNITY and DOCUMENTATION, which each have titles which are sufficiently descriptive to be found on their own.

File: content/help.html (hunk 1 above)
Comment (brian-brazil) [inline content/help.html]: Fixed.

**Comment by brian-brazil** [comment]
@grobie I'm following what the community page was doing.

@jrv I see your concern, this is more task oriented as a new user would have to guess which of docs/community would be of assistance - I personally always hate having to reverse engineer a how a website is laid out when looking for basic information.

**Comment by juliusv** [comment]
So personally I still feel that the extra visual overhead isn't worth the gains. An approach that would work without adding an extra menu item for mainly duplicated/pointer content would be much preferable IMO.

If others do think it'd be worth adding another menu point just for this, I'd suggest moving it next to "BLOG" since that is also four letters and looks less awkward than if one small word is squeezed between two larger ones.

**Pull request closed without merging**